meinberlin/assets/scss/components/_document.scss

```diff
@@ -11,6 +11,7 @@
 }
 
 .paragraph__content {
+    @include clearfix;
```

Comment:
@goapunk it would be better if we could give all ckeditor rendered content a class that we could assign this to?